metadata-ws/src/test/resources/features/security.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Security related tests
+
+  Scenario: access security disabled links
+    When I request ananymous GET /
+    Then the response code should be 200
+    When I request ananymous GET /swagger-ui.html
+    Then the response code should be 200
+
+
+  Scenario: update security enabled links
+    Given I request POST /publications with JSON payload:
+    """
+    {
+      "publicationId": "Publication5"
+    }
+    """
+    And set the URL to PUBLICATION5
+    Then the response code should be 201
+
+    When I request PUT PUBLICATION5 with JSON payload:
+    """
+    {
+      "publicationId": "Publication7"
+    }
+    """
+    Then the response code should be 204
+    When I request GET with value of PUBLICATION5
+    Then the response code should be 200
+    And the response should contain field publicationId with value Publication7
+
+    When I request DELETE with value of PUBLICATION5
+    Then the response code should be 204
+
+
+  Scenario: access security enabled links and fail
+    Given I set authorization with testuser having default role
+    When I request authority set POST /publications with JSON payload:
```

Comment:
```suggestion
    When I request POST /publications with JSON payload:
```

Reply:
I have set implicitly `testuser` with default role for all read operations and `testoperator` with SERVICE_OPERATOR role for update operations. Either i can document this or i can change entire set of statements explicitly setting user and role. 
Please let me know
For security feature i am setting role and requesting operation

Reply:
u have already provided in the "Given", I guess it is redundant in  "When"

Reply:
Existing statements have `When I request POST ...` To differentiate i have used  `When I request authority set POST ...`.  
Otherwise we can change all the existing statements to include "Given I set authorization with ..."

Reply:
yes actually u are setting authorization for all request so, so you should include Given

Reply:
sure, with entire feature set(in all files), i will include Given with authorization. Should look something like
`Given I set authorization with testoperator having SERVICE_OPERATOR role`
`When I request authority set POST /publications with JSON payload`
.....

Reply:
I'm happy with the way authorisation is specified in test scenarios. @selvaebi If you agree, we can mark this as resolved